go.mod

```diff
@@ -42,7 +42,7 @@ replace (
 	k8s.io/client-go => k8s.io/client-go v0.19.7
 	k8s.io/code-generator => k8s.io/code-generator v0.19.7
 	k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20200805222855-6aeccd4b50c6
-	knative.dev/eventing => github.com/openshift/knative-eventing v0.99.1-0.20210406161751-80f49379d45d
-	knative.dev/eventing-kafka => github.com/openshift-knative/eventing-kafka v0.19.1-0.20210420075609-dbcba4bee3ce
+	knative.dev/eventing => github.com/cardil/eventing v0.11.1-0.20210525135852-116524fc7b9f
+	knative.dev/eventing-kafka => github.com/cardil/eventing-kafka v0.21.1-0.20210525161113-e17d14cc6e79
```

Comment:
/hold

Until https://github.com/openshift-knative/eventing-kafka/pull/215 and https://github.com/openshift/knative-eventing/pull/1284 are merged.

Reply:
/unhold

Both openshift-knative/eventing-kafka#215 and openshift/knative-eventing#1284 are now merged. As well as https://github.com/openshift/release/pull/18852